path.hpp

```diff
@@ -0,0 +1,221 @@
+// The path class.
+//
+// Copyright (C) 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020 Gregory W. Chicares.
```

Comment:
```suggestion
// Copyright (C) 2020 Gregory W. Chicares.
```
(AFAICS this is a new file)